apps/system/js/app_chrome.js

```diff
@@ -110,7 +110,8 @@
     var className = this.CLASS_NAME + this.instanceID;
 
     return `<div class="chrome chrome-combined" id="${className}">
-              <gaia-progress></gaia-progress>
+              <gaia-progress role="progressbar" aria-live="polite">
```

Comment:
I would have these attributes set automatically in the bootstrap of gaia-progress.